### PR TITLE
Added table of standard error codes to end of custom error codes topic.

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

To inform user about hardcoded standard error codes.

## Changes

Added table to end of custom errors topic.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

File: source/shared/smart-contracts/guides/custom-errors.rst [hunk 1]
```diff
@@ -55,3 +55,25 @@ Return custom errors, as you would with any other error type:
...
+The following standard error codes exist:
+
+=========================================== ===========================
+Variant                                     Error code
+=========================================== ===========================
+()	                                         i32::MIN + 1 (-2147483647)
+ParseError	                                i32::MIN + 2 (-2147483646)
+LogError::Full	                             i32::MIN + 3 (-2147483645)
+LogError::Malformed	                       i32::MIN + 4 (-2147483644)
+NewContractNameError:://! MissingInitPrefix i32::MIN + 5 (-2147483643)
```
Comment (abizjak) [inline source/shared/smart-contracts/guides/custom-errors.rst:71]:
```suggestion
NewContractNameError::MissingInitPrefix                 i32::MIN + 5 (-2147483643)
```

**Review by abizjak — CHANGES_REQUESTED** [review]
Either we need to include links to the errors like they are here https://docs.rs/concordium-std/1.0.0/concordium_std/ Here are the links https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/concordium-std/src/lib.rs#L130 (see section below as well) https://github.com/Concordium/concordium-rust-smart-contracts/blob/main/concordium-std/src/lib.rs#L145

Can you insert these links or do you want me to do it?

I think I should just update the documentation

File: source/shared/smart-contracts/guides/custom-errors.rst (hunk 1 above)
Comment (dg-concordium) [reply source/shared/smart-contracts/guides/custom-errors.rst:71]: You are welcome to update the documentation to include the links to the errors. I was simply inserting what you added to the original issue.

**Comment by Bargsteen** [comment]
What is the status of this PR?

**Comment by dg-concordium** [comment]
> What is the status of this PR?

Unfinished. It was awaiting Smart Contracts v1.

**Comment by abizjak** [comment]
I think someone (I guess me?) should get this to completion finally.

I completely forgot about it. Thanks for the reminder @dg-concordium

**Review by limemloh — APPROVED** [review]

**Review by DOBEN — APPROVED** [review]

**Pull request merged**